samza-core/src/main/java/org/apache/samza/runtime/ApplicationRunnerMain.java

```diff
@@ -50,8 +50,12 @@ public static void main(String[] args) throws Exception {
     ApplicationRunnerCommandLine cmdLine = new ApplicationRunnerCommandLine();
     OptionSet options = cmdLine.parser().parse(args);
     Config orgConfig = cmdLine.loadConfig(options);
-    Config config = Util.rewriteConfig(orgConfig);
     ApplicationRunnerOperation op = cmdLine.getOperation(options);
+    invokeApplicationRunner(orgConfig, op);
+  }
+
+  public static ApplicationRunner invokeApplicationRunner(Config orgConfig, ApplicationRunnerOperation op) {
```

Comment:
Do you need to return the ApplicationRunner?

Nitpick: s/orgConfig/config/

Reply:
It will be needed to get the status.